SystemReference/Payload/Camera/CMakeLists.txt

```diff
@@ -7,20 +7,27 @@
 # Note: using PROJECT_NAME as EXECUTABLE_NAME
 ####
 set(SOURCE_FILES
-		"${CMAKE_CURRENT_LIST_DIR}/Camera.fpp"
-		"${CMAKE_CURRENT_LIST_DIR}/Camera.cpp"
-		)
+	"${CMAKE_CURRENT_LIST_DIR}/Camera.fpp"
+	"${CMAKE_CURRENT_LIST_DIR}/Camera.cpp"
+)
 register_fprime_module()
-if (DEFINED SYSTEM_REFERENCE_OPENCV_PATH AND NOT DEFINED FPRIME_PRESCAN)
-	get_module_name("${CMAKE_CURRENT_LIST_DIR}")
-	target_link_libraries(${MODULE_NAME} PUBLIC ${OpenCV_LIBS})
-endif()
+
+find_package(PkgConfig)
+
+pkg_check_modules(LIBCAMERA REQUIRED IMPORTED_TARGET libcamera)
```

Comment:
Is it possible to build `libcamera` here?  Or do we always have a two part interaction?

Reply:
Move CMAKE standard version here

Reply:
Building libcamera here will be done in future PR